vnext/ReactUWP/Views/Image/ReactImage.cpp

```diff
@@ -97,9 +97,10 @@ namespace react {
             winrt::LoadedImageSurface::StartLoadFromStream(memoryStream) :
             winrt::LoadedImageSurface::StartLoadFromUri(uri) };
 
-          surface.LoadCompleted({ this, &ReactImage::LoadedImageSurfaceHandler });
-
-          m_brush->Source(surface);
+          // using a lambda here to capture the surface and prevent it from freeing while the image loads
+          surface.LoadCompleted([this, surface](winrt::LoadedImageSurface const& sender, winrt::LoadedImageSourceLoadCompletedEventArgs const& args) {
+			      this->LoadedImageSurfaceHandler(sender, args);
```

Comment:
>this->LoadedImageSurfaceHandler(sender, args); [](start = 9, length = 46)

Delete LoadedImageSurfaceHandler and move the code to the lamba.
You can then use the captured surface to set the brush source if you want. 
Either way, please get rid of the sad/unnecessary "sender.as<winrt::LoadedImageSurface>()" I left behind :)